client/src/features/project/ProjectEntityHeader.tsx

```diff
@@ -27,7 +27,6 @@ type ProjectEntityHeaderProps = EntityHeaderProps & { isInKg: boolean };
 function ProjectEntityHeader(props: ProjectEntityHeaderProps) {
   const { fullPath } = props;
 
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars, no-unused-vars
   const { data: _kgData } = useProjectMetadataQuery(
```

Comment:
If the data is not used:

```suggestion
useProjectMetadataQuery(
```